Sources/UIAlertControllerView.swift

```diff
@@ -97,9 +166,18 @@ class UIAlertControllerButton: Button {
         }
     }
 
+    var titleLabelOriginX: CGFloat?
+    
     override func layoutSubviews() {
         super.layoutSubviews()
+        if let titleLabelOriginX = titleLabelOriginX {
+            titleLabel?.frame.origin.x = titleLabelOriginX
+        }
+    }
+}
```

Comment:
I guess instead of making it an optional - could make it `0` by default.

```suggestion
    var titleLabelOriginX: CGFloat = 0
    
    override func layoutSubviews() {
        super.layoutSubviews()
        titleLabel?.frame.origin.x = titleLabelOriginX
    }
}
```

Reply:
I think this won't work because setting `Button.contentHorizontalAlignment` wouldn't have any effect anymore because the label will always start at 0.